dist/images/Makefile

```diff
@@ -43,7 +43,7 @@ fedora-dev: bld
 	${OCI_BIN} build \
 		     --build-arg OVS_BRANCH=$(OVS_BRANCH) \
 		     --build-arg OVN_BRANCH=$(OVN_BRANCH) \
-		     -t ovn-kube-fedora-dev -f Dockerfile.fedora.dev .
+		     -t ovn-kube-fedora-dev -f Dockerfile.fedora .
```

Comment:
What's the reason for this change?

Reply:
+1 seems unrelated and should probably be removed

Reply:
ah missed that. [Changed back](https://github.com/ovn-kubernetes/ovn-kubernetes/compare/125ae3a6ffee572e8ea7209737b293f4501d48f1..fd5cfa9dc126a9ef3d1bd64780001bc3405d9bca)

Reply:
@qinqon what do we do with this ?...

Reply:
Fixed